From cc328b43ebb9a54a256ec73a86fb18381b2038e6 Mon Sep 17 00:00:00 2001
From: Eli Schwartz <eschwartz@archlinux.org>
Date: Tue, 11 Jan 2022 21:57:26 -0500
Subject: [PATCH] fix duplicate languages causing meson 0.61 to error out

It tried to create multiple targets with the same name and generating
the same file. On older versions of meson, this was a run_target and
just ran twice, but was still wrong.
---
 help/LINGUAS | 1 -
 1 file changed, 1 deletion(-)

diff --git a/help/LINGUAS b/help/LINGUAS
index 68f089c4..5352eecd 100644
--- a/help/LINGUAS
+++ b/help/LINGUAS
@@ -6,7 +6,6 @@ es
 fr
 hu
 id
-id
 lv
 pt_BR
 sv
-- 
GitLab

